librosa/core/pitch.py

```diff
@@ -619,6 +623,9 @@ def pyin(y, sr=22050, frame_length=2048, win_length=1024, hop_length=None, fmin=
     f0: np.ndarray [shape=(n_frames,)]
         time series of fundamental frequencies in Hertz.
 
+    voiced_flag: np.ndarray [shape=(n_frames,)]
+        time series containing boolean flags indicating whether a frame is voiced or not.
+
     voiced_prob: np.ndarray [shape=(n_frames,)]
         time series containing the probability that a frame is voiced.
 
```

Comment:
Example strings need a new-line between text description and the start of code:

```
Examples
-------------
Description

>> some_code()
```

This example should be updated to use the required parameters for pyin (fmin, fmax).

It'd be great if it could also plot the results over a spectrogram; maybe check the example for [`spectral_bandwidth`](https://librosa.org/doc/main/generated/librosa.feature.spectral_bandwidth.html#librosa.feature.spectral_bandwidth) for a starting point on that.